hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoTokenHandlerTestBase.java

```diff
@@ -168,6 +169,10 @@ public class CryptoTokenHandlerTestBase {
             .build();
     protected List<CustomFee> customFees = List.of(withFixedFee(fixedFee), withFractionalFee(fractionalFee));
 
+    protected TokensConfig tokensConfig = new TokensConfig(
```

Comment:
We should use the config API (+ https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-config/src/testFixtures/java/com/hedera/node/config/testfixtures/HederaTestConfigBuilder.java) to create it

Reply:
Changed . Thanks

Reply:
Have you pushed? I still see the manual record creation

Reply:
Pushed